### DISCUSSION TITLE
"Edit this page" buttons in the Docs site lead to a 404

### DISCUSSION THREAD

**Discussion by jamietanna** [renovatebot/renovate#19310] [Suggest an Idea]
When browsing to a page on the documentation site and clicking ✏️, the "Edit this page" button, we receive a 404.

For instance, on https://docs.renovatebot.com/modules/platform/github/#github-and-github-enterprise-server, we click it and go to https://github.com/renovatebot/renovate/edit/main/docs/usage/modules/platform/github/index.md which is a 404.

The correct link is:

```diff
-https://github.com/renovatebot/renovate/edit/main/docs/usage/modules/platform/github/index.md
+https://github.com/renovatebot/renovate/edit/main/lib/modules/platform/github/index.md
```

I can't immediately see how to resolve this, but I believe it's due to the way that we build + copy everything into the `docs` folder.

**Comment by viceice** [comment]
- reopened #19309

needs changes to our docs generator to override `edit_url` metadata for generated pages.
